opensrp-reveal/src/main/java/org/smartregister/reveal/widget/GeoWidgetFactory.java

```diff
@@ -126,7 +129,9 @@ public void onMapReady(MapboxMap mapboxMap) {
                 if (finalOperationalArea != null) {
                     mapboxMap.setCameraPosition(mapboxMap.getCameraForGeometry(Geometry.fromJson(finalOperationalArea)));
                 } else {
-                    mapView.focusOnUserLocation(true);
+                    String bufferRadiusStr = RevealApplication.getInstance().getGlobalConfigs().get(LOCATION_BUFFER_RADIUS_IN_METRES);
+                    Float bufferRadius = bufferRadiusStr == null ? DEFAULT_LOCATION_BUFFER_RADIUS_IN_METRES : Float.valueOf(bufferRadiusStr);
+                    mapView.focusOnUserLocation(true, bufferRadius);
```

Comment:
Make a common function in Utils so that line 132-124 is not repeated in the GeowidgetFactory

Reply:
Done.